app/models/spotlight/search.rb

```diff
@@ -82,7 +82,7 @@ def search_builder
     end
 
     def repository
-      Blacklight.default_index
+      @repository ||= Blacklight::Solr::Repository.new(blacklight_config)
```

Comment:
I don't think we necessarily want to hard-code `Blacklight::Solr` here, although `Blacklight.repository_class` seems to be Blacklight 6-only. @jcoyne, does it make sense to add that method to Blacklight 5.x with a hard-coded default, or just remember to revisit this when go to Blacklight 6?